diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -361,4 +434,59 @@ mod tests {
         let query = select(datetime("9999-01-08 00:00:00.000000").eq(distant_future));
         assert!(query.get_result::<bool>(connection).unwrap());
     }
+
+    #[test]
+    fn datetime_with_tz_encodes_correctly() {
+        let connection = &mut connection();
+        let time = NaiveDate::from_ymd(1970, 1, 1).and_hms(0, 0, 0);
+        let query = select(datetime_with_tz("1970-01-01 00:00:00.000000Z").eq(time));
+        assert_eq!(Ok(true), query.get_result(connection));
+    }
```

Comment:
I don't think I got these working but was curious (sorry a very n00b question) how to run individual tests -- I tried the following but couldn't figure out what to do:

```
@ diesel [stevenchu]$ cargo test --package diesel --features sqlite -- sqlite::types::date_and_time::chrono --nocapture
    Finished test [unoptimized + debuginfo] target(s) in 0.02s
     Running unittests (target/debug/deps/diesel-91bdefc62c2d4481)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 26 filtered out; finished in 0.00s

   Doc-tests diesel

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 171 filtered out; finished in 0.16s
```